frontend/src/services/password-thunk.js

```diff
@@ -134,3 +134,8 @@ export const uploadBackupThunk = createAsyncThunk(
     }
   }
 );
+
+export const clearEncryptData = createAsyncThunk(
```

Comment:
This can be a reducer, does not need to be an async thunk

Reply:
should i create it  as 

export const clearEncryptData = createAction('password/clearEncryptData');  
 in the source code .

Reply:
Nope right here: https://github.com/frankhart2018/primistore/blob/b77ff2dc877c0b4e1f0f70b2f217179efabb803f/frontend/src/reducers/password-reducer.js#L41. Here's an example: https://github.com/frankhart2018/cs-5610-404-not-found/blob/08de309606e095cd1937f24185709968fffc8bf3/frontend/src/reducers/file-reducer.js#L17.